cl/recap/mergers.py

```diff
@@ -954,10 +954,10 @@ async def add_docket_entries(
             await duplicate_rd_queryset.exclude(pk=rd.pk).adelete()
 
         rd.pacer_doc_id = rd.pacer_doc_id or docket_entry["pacer_doc_id"]
-        description = docket_entry.get("short_description") or rd.description
```

Comment:
The issue was that this `rd.description` would in some cases correspond to an `RECAPDocument.ATTACHMENT`, so the attachment description would get cached here and subsequently replace the `RECAPDocument.PACER_DOCUMENT` description, the description wasn't getting deleted.